client/state/ui/selectors.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Returns the site object for the currently selected site.
+ *
+ * @param  {Object} state  Global state tree
+ * @return {Object}        Selected site
+ */
+export function getSelectedSite( state ) {
+	if ( ! state.ui.selectedSite ) {
+		return null;
+	}
+
+	return state.sites.byId[ state.ui.selectedSite ];
```

Comment:
Maybe `getSelectedSite` should receive two params with `state.sites` and `state.ui.selectedSite` instead of the entire `state` object?